admin/error-handling/class-webchangedetector-error-handler.php

```
@@ -0,0 +1,347 @@
+	 * Register default error callbacks.
+	 */
+	private function register_default_callbacks() {
+		// API error callback.
+		$this->register_error_callback( self::CATEGORY_API, function( $exception, $category, $attempt ) {
+			// Clear any cached API responses on API errors.
+			wp_cache_delete( 'wcd_api_response', 'webchangedetector' );
+		} );
+
+
+		// Critical error callback.
+		$this->register_error_callback( self::SEVERITY_CRITICAL, function( $exception, $category, $attempt ) {
```

PHPMD (2.15.0)

Avoid unused parameters such as '$attempt'. (Unused Code Rules)

(UnusedFormalParameter)